### ISSUE TITLE
🐛 Biome server crashed on VSCODE after migrating from 1.9 to 2.0

### ISSUE THREAD

**Issue by gpaulini** [biomejs/biome#7195]
### Environment information

```block
CLI:
  Version:                      2.0.6
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.17.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/10.9.2

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false

Workspace:
  Open Documents:               0
```

### What happened?

pnpm upgrade v1.9.4 to v2.0.6 -> config migration -> biome restart on vscode

2025-08-12 10:06:40.967 [info] [Error - 10:06:40 AM] Server process exited with code 0.
2025-08-12 10:06:40.969 [info] [Info  - 10:06:40 AM] Connection to server got closed. Server will restart.
2025-08-12 10:06:40.969 [info] true
2025-08-12 10:06:41.078 [info] Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_service/src/settings.rs:78:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.

2025-08-12 10:06:41.091 [info] [Error - 10:06:41 AM] Server process exited with code 0.
2025-08-12 10:06:41.094 [info] [Error - 10:06:41 AM] The biome server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.

### Expected result

To work properly

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by gpaulini** [comment]
Nevermind, I restarted vscode and it worked like a charm.
